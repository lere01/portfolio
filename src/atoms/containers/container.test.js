import React from 'react';
import renderer from 'react-test-renderer';
import Container from './container';

it("renders correctly when corrected", () => {
    const tree = renderer
        .create(<Container />)
        .toJSON();

    expect(tree).toMatchSnapshot();
})