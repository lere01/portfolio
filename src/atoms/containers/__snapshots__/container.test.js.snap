// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when corrected 1`] = `
<div
  className=""
>
   
   
</div>
`;
